dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -441,30 +450,42 @@ private void selectMultipleItems(final List<String> rowNameList, final TreeTable
-            ds3CancelSingleJobTask.setOnFailed(SafeHandler.logHandle(event -> LOG.error("Failed to cancel job")));
-            ds3CancelSingleJobTask.setOnSucceeded(SafeHandler.logHandle(event -> {
-                LOG.info("Get Job cancelled");
-                loggingService.logMessage("GET Job Cancelled", LogType.INFO);
-                refreshFileTreeView();
-            }));
-            workers.execute(ds3CancelSingleJobTask);
-        }));
-        jobWorkers.execute(getJob);
+            final Path localPath, final Session session) {
+        listFiles.stream()
+                .map(Ds3TreeTableValueCustom::getBucketName)
+                .distinct()
+                .forEach(bucket -> {
+                    final ImmutableList<kotlin.Pair<String, String>> fileAndParent = listFiles.stream()
+                            .filter(ds3TreeTableValueCustom -> Objects.equals(ds3TreeTableValueCustom.getBucketName(), bucket))
+                            .map(ds3 -> new kotlin.Pair<>(
+                                    ds3.getFullName(),
+                                    ds3.getParent()))
+                            .collect(GuavaCollectors.immutableList());
+                    final JobTaskElement jte = new JobTaskElement(settingsStore, loggingService, dateTimeUtils, session.getClient(), jobInterruptionStore, savedJobPrioritiesStore);
+                    final GetJobData getJobData = new GetJobData(fileAndParent, localPath, bucket, jte);
+                    final JobTask jobTask = new JobTask(new GetJob(getJobData));
+                    jobTask.setOnSucceeded(SafeHandler.logHandle(event -> {
+                        LOG.info("Get Job completed successfully");
+                        refreshFileTreeView();
+                    }));
+                    jobTask.setOnFailed(SafeHandler.logHandle(event -> {
+                        final Throwable exception = event.getSource().getException();
+                        LOG.error("Get Job failed", exception);
+                        loggingService.logMessage("Get Job failed with message: " + exception.getMessage(), LogType.ERROR);
+                        refreshFileTreeView();
+                    }));
+                    jobTask.setOnCancelled(SafeHandler.logHandle(cancelEvent -> {
+                        final Ds3CancelSingleJobTask ds3CancelSingleJobTask = new Ds3CancelSingleJobTask(jobTask.getJobId().toString(), endpointInfo, jobInterruptionStore, JobRequestType.GET.toString(), loggingService);
+                        ds3CancelSingleJobTask.setOnFailed(SafeHandler.logHandle(event -> LOG.error("Failed to cancel job")));
+                        ds3CancelSingleJobTask.setOnSucceeded(SafeHandler.logHandle(event -> {
+                            LOG.info("Get Job cancelled");
+                            loggingService.logMessage("GET Job Cancelled", LogType.INFO);
```

Comment:
when a put job was canceled you used LogType of success and here you are using Info, is that by design?

Reply:
I think this is indicates that the cancel was successful. Would Success be a better choice?

Reply:
I think it should be the same on all canceled operation PUTs and GUTs,
and BTW is cancelled misspelled? should it be canceled?